### PR TITLE
Chore: Try to fix flaky reverse proxy test

### PR DESCRIPTION
**Pull request by marefr**
**What this PR does / why we need it**:
Should fix flaky test failure like [this](https://drone.grafana.net/grafana/grafana/58815/1/12). 

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: pkg/util/proxyutil/reverse_proxy_test.go [hunk 1]
```diff
@@ -101,6 +102,7 @@ func TestReverseProxy(t *testing.T) {
 			{
 				desc:               "Timed out request should return 504 Gateway timeout",
 				transport:          timedOutTransport,
+				responseWaitTime:   500 * time.Millisecond,
```
Comment (sakjur) [inline pkg/util/proxyutil/reverse_proxy_test.go:105]: Will we actually wait 500 milliseconds here? It seems a bit excessive, 100x should do and would slow down the tests less.

**Review by sakjur — APPROVED** [review]
Not familiar with the particulars, but the changes seems reasonable enough

File: pkg/util/proxyutil/reverse_proxy_test.go (hunk 1 above)
Comment (marefr) [reply pkg/util/proxyutil/reverse_proxy_test.go]: Good point. Fixed

**Review by wbrowne — APPROVED** [review]
🤞

**Pull request merged**